rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json

```diff
@@ -282,6 +358,113 @@
         }
       }
     },
+    "issuance": {
+      "title": "Mode of issuance",
+      "type": "object",
+      "required": [
+        "maintype",
+        "subtype"
+      ],
+      "propertiesOrder": [
+        "maintype",
+        "subtype"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "maintype": {
```

Comment:
Same as other comment of @AoNoOokami: `mainType` or `main_type`?
https://github.com/rero/developer-resources/blob/master/guidelines/json-schemas.md#key-names